localstack/services/lambda_/invocation/lambda_models.py

```diff
@@ -41,7 +41,7 @@
 from localstack.constants import AWS_REGION_US_EAST_1
 from localstack.services.lambda_.api_utils import qualified_lambda_arn, unqualified_lambda_arn
 from localstack.utils.archives import unzip
-from localstack.utils.strings import long_uid
+from localstack.utils.strings import long_uid, short_uid
```

Comment:
@giograno Does this hack invalidate all Lambda CloudPods due to a changed import 🤔